src/api/ledgerStorage/github.js

```diff
@@ -42,17 +63,18 @@ export class GithubStorage implements LedgerStorage {
         variables: {
           owner: this._repoId.owner,
           repo: this._repoId.name,
+          expression: `${this._branch}:data/ledger.json`,
         },
       }),
     };
 
-    const res = await fetch(`https://api.github.com/graphql`, opts);
+    const res = await fetch(`${GithubStorage.ENDPOINT}/graphql`, opts);
```

Comment:
nit: 'res' and 'opts' are still more abbreviated that I'd normally go for, but not so bad